pkg/reconciler/apiserversource/apiserversource.go

```diff
@@ -100,7 +96,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, key string) error {
 	// Convert the namespace/name string into a distinct namespace and name
 	namespace, name, err := cache.SplitMetaNamespaceKey(key)
 	if err != nil {
-		r.Logger.Errorf("invalid resource key: %s", key)
+		logging.FromContext(ctx).Error("invalid resource key", zap.Any("key", key))
```

Comment:
`zap.Any("key", key)` is redundant, the logger already injects `"knative.dev/key"`

Reply:
Thanks! I also fixed it  in cronjobsource.go